public/index.html

```diff
@@ -16,6 +16,7 @@
         </div>
       </div>
     </main>
+    <script src="/bower_components/fxa-relier-client/fxa-relier-client.min.js"></script>
```

Comment:
Sure, I can remove direct purchase sign-in for now. We can always add it in later.